Jenkinsfile

```diff
@@ -257,6 +257,7 @@ pipeline {
           steps {
             sconsBuild target: 'cart',
                        scons_local_replace: true,
+                       USE_INSTALLED: 'none',
```

Comment:
I'm concerned about the need to alter this default behaviour of `sconsBuild` for this patch.

Do we understand why it's needed and whether that is going to affect the `sconsBuild` use in the daos and cart `Jenkinsfile`s?

Reply:
This patch fixed a bug with USE_INSTALLED handling.   So now, because 'all' is the default, the ofi build gets configured without $PSM2_PREFIX and therefore can't find psm2